mcs/class/Mono.Debugger.Soft/Mono.Debugger.Soft/ArrayMirror.cs

```diff
@@ -92,6 +92,14 @@ public class ArrayMirror : ObjectMirror, IEnumerable {
 			vm.conn.Array_SetValues (id, index, vm.EncodeValues (values));
 		}
 
+		public void SetByteValues (byte [] bytes)
+		{
+			if (bytes != null && bytes.Length != Length) {
```

Comment:
This might need a protocol version check.

Reply:
This implementation uses this CMD_ARRAY_REF_SET_VALUES that exists since you write it :), I don't think that we need a protocol version check.